integrations/object_store/src/service/deleter.rs

```diff
@@ -0,0 +1,70 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+use std::sync::Arc;
+
+use futures::stream::{self, StreamExt};
+use object_store::ObjectStore;
+use opendal::raw::oio::BatchDeleteResult;
+use opendal::raw::*;
+use opendal::*;
+
+use super::error::parse_error;
+
+pub struct ObjectStoreDeleter {
+    store: Arc<dyn ObjectStore + 'static>,
+}
+
+impl ObjectStoreDeleter {
+    pub(crate) fn new(store: Arc<dyn ObjectStore + 'static>) -> Self {
+        Self { store }
+    }
+}
+
+impl oio::BatchDelete for ObjectStoreDeleter {
+    async fn delete_once(&self, path: String, _: OpDelete) -> Result<()> {
+        let object_path = object_store::path::Path::from(path);
```

Comment:
We have seen many `object_store::path::Path`, I think we can import them as `ObjectStorePath` for better reading.

Reply:
fixed in f6b905fd5a